src/Arcus.Templates.WebApi/ExampleProviders/HealthReportExampleProvider.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+using Swashbuckle.AspNetCore.Filters;
+
+namespace Arcus.Templates.WebApi.ExampleProviders
```

Comment:
I also wonder how we can prevent this file from being added / generated when the open-api parameter is set to false when creating a new API based on this template ?

Reply:
@stijnmoreels can help on this

Reply:
In the meantime, I've done it by explicitly including the file in the csproj file and specified a condition attribute.  Don't know if this is the preferred way @stijnmoreels  ?

Reply:
Yeah, you shouldn't alter the `.csproj` file for this. You should rather look into the `./.template.config/template.json` file instead. There you'll find a section `"modifiers"` where you can exclude files based on flags. There's already such a mofidier configured to removed/add the `appsettings.json` file from the project.
Let me know if you experience any problems.

Reply:
Ok, I've tried it.  Is there a way to test this locally, to see if I've done this correctly ?